src/components/Onfido/BaseOnfidoWeb.tsx

```diff
@@ -78,17 +81,15 @@ function initializeOnfido({sdkToken, onSuccess, onError, onUserExit, preferredLo
                 },
             },
         ],
-        smsNumberCountryCode: CONST.ONFIDO.SMS_NUMBER_COUNTRY_CODE.US,
-        showCountrySelection: false,
```

Comment:
Was removed some time ago https://documentation.onfido.com/sdk/web/#change-in-sdk-options